CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v33.0.0...main)
 
+* Android
+  * Update the Kotlin Gradle Plugin to version 1.4.10.
```

Comment:
For my own understanding: this means we compile and test against that version of Kotlin and don't promise any lower Kotlin version still works.
But consumers _could_ compile their own code against other Kotlin versions and still rely on us?

Reply:
Yes, that's my understanding as well

Reply:
I double checked with @pocmo and he confirmed this. This is also discussed in the [Kotlin Components Stability](https://kotlinlang.org/docs/reference/evolution/components-stability.html) page!